internal/host/validator.go

```diff
@@ -251,6 +251,10 @@ type validator struct {
 	providerRegistry registry.ProviderRegistry
 }
 
+func (v *validator) isMediaConnected(c *validationContext) ValidationStatus {
```

Comment:
Maybe this validation should be part of isConnected, and not a different function? In all your changes you are check IfConnect and IfMediaConnected together, so why not unite them into one function, since they also share the same state?

Reply:
No, not strongly opposed :)